packages/nuxt/README.md

```diff
@@ -126,18 +126,6 @@ other imports:
 }
 ```
 
-If you are getting an `import-in-the-middle` error message, add the package with a minimum version of `1.10.0` as a
```

Comment:
@s1gr1d I removed this note because we now pin IITM versions in `@sentry/node`, so folks should not run into stale versions anymore.

Reply:
Thanks! I tested this and yes, it works :)